tests/wpt/web-platform-tests/fetch/nosniff/stylesheet.html

```diff
@@ -3,28 +3,36 @@
 <script src=/resources/testharnessreport.js></script>
 <div id=log></div>
 <script>
-  var fails = ["", "?type=", "?type=x", "?type=x/x"],
-      passes = ["?type=text/css", "?type=text/css;blah"]
+  var fails = [null, "", "x", "x/x"],
+      passes = ["text/css", "text/css;charset=utf-8"]
```

Comment:
Hm, maybe! My memory said that broken parameters should just be ignored, but media types are spread over so many RFCs that I can't find that anywhere.